src/utils/MavenUtil.ts

```diff
@@ -165,3 +189,23 @@ export function findChildMavenModules(xmlString: string): Map<string, string[]>
     });
     return childrenMap;
 }
+
+/**
+ * Return true if the liberty-maven-plugin version is compatible
+ * for dev mode with containers 
+ * 
+ * @param plugin JS object for liberty-maven-plugin
+ */
+function mavenPluginVersionValid(plugin: any): boolean {
```

Comment:
Instead of naming this as "Valid", could use a more descriptive name to indicate that it relates to containers.

Same applies to Gradle.